src/Engine.php

```diff
@@ -53,7 +53,7 @@
 use Infection\Process\Runner\TestRunConstraintChecker;
 use Infection\TestFramework\Coverage\CoverageDoesNotExistException;
 use Infection\TestFramework\Coverage\XmlReport\PhpUnitXmlCoverageFactory;
-use Infection\TestFramework\HasExtraNodeVisitors;
+use Infection\TestFramework\IgnoresAdditional;
```

Comment:
`IgnoresAdditional` -> `IgnoresAdditionalNodes`? As it not quite clear what we should ignore on the Test Framework level